src/frontend/screens/Library/components/SteamInstall/SteamInstallDialog.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { TFunction } from 'i18next'
+import { Dialog } from 'frontend/components/UI/Dialog'
+import { hasProgress } from 'frontend/hooks/hasProgress'
+import { faSteam, faWindows } from '@fortawesome/free-brands-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import './index.scss'
+
+interface SteamInstallDialogProps {
+  isInstalling: boolean
+  onClose: () => void
+  onInstall: () => void
+}
+
+const getProgressMessage = (percent: number, t: TFunction) => {
+  if (!percent) {
+    return t('Please Wait', 'Please Wait')
+  }
+  if (percent > 95) {
+    return t('Installing', 'Installing')
+  }
+  return t('Downloading', 'Downloading {{percent}}%', {
+    percent: percent.toFixed(0)
+  })
+}
+
+const SteamInstallDialog: React.FC<SteamInstallDialogProps> = ({
+  isInstalling,
+  onClose,
+  onInstall
+}) => {
+  const { t } = useTranslation()
+  const [progress] = hasProgress('steam')
+
+  const percent = progress.percent ?? 0
+  const downloadMessage = getProgressMessage(percent, t)
+
+  return (
+    <Dialog showCloseButton={false} onClose={onClose} className="steamModal">
+      <div>
+        <h6>
+          <span className="iconsWrapper">
+            <FontAwesomeIcon icon={faSteam} />
+            <FontAwesomeIcon icon={faWindows} />
+          </span>
+          {t('Steam Installation')}
+        </h6>
+      </div>
+      <div>
+        <ul>
+          <li>
+            {t(
+              'Recommended specs for a better experience',
+              'Recommended specs for a better experience'
```

Comment:
these `t` calls with the key being the same as the english string doesn't match what we do in the rest of the app